test/controller.js

```diff
@@ -301,4 +301,76 @@ describe('Integration tests', function () {
         assert.isTrue(res.writeHead.calledWith(123));
 
     });
+    
+    it('should provide a way to delete one response for a given endpoint', function () {
```

Comment:
this test does not test the behaviour.. it just make sure we are calling the right method.